std-bits/table/src/main/java/org/enso/table/data/column/builder/object/Builder.java

```diff
@@ -1,6 +1,7 @@
 package org.enso.table.data.column.builder.object;
 
 import org.enso.table.data.column.storage.Storage;
+import org.graalvm.polyglot.Value;
```

Comment:
Guessing not needed as not used.

Reply:
Should the appends be taking Value no Object now?
Probably a separate PR is so.

Reply:
Not necessarily, it depends where we want to handle the polyglot boundary. Appends can be working with Objects as they are also used by pure Java methods, like Delimited parsing. We could wrap there to Values using `Value.asValue` but IMO it would feel artificial. I was thinking of this, but decided it makes sense to keep them as-is. Instead, we should be handling the conversions at places where we are getting values from Enso - which is what I tried to do here.

So I'd keep it as is, but we can revisit if needed - but that would indeed be a separate task.